package.json

```diff
@@ -157,6 +157,7 @@
     "jest-canvas-mock": "2.5.2",
     "jest-environment-jsdom": "29.6.2",
     "jest-preset-angular": "13.1.1",
+    "ng-packagr": "^16.2.3",
```

Comment:
Maybe we could also add a script command to the `package.json`? Btw when running `generate-package.js` I noticed that it needs to be called from inside `package` and not from root.

Reply:
good point, this should be improved

Reply:
As for adding a script to publish the package in the root `package.json`, this could definitely be useful; I think we can do this later on, if e.g. we want to publish the package automatically on every commit/tag.